src/SurveyService.Validation/Question/CreateQuestionRequestValidator.cs

```diff
@@ -0,0 +1,46 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.SurveyService.Data.Interfaces;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Question;
+using LT.DigitalOffice.SurveyService.Validation.Question.Interfaces;
+using System;
+
+namespace LT.DigitalOffice.SurveyService.Validation.Question;
+
+public class CreateQuestionRequestValidator : AbstractValidator<CreateSingleQuestionRequest>, ICreateQuestionRequestValidator
+{
+  public CreateQuestionRequestValidator(
+    IQuestionRepository _questionRepository
+    )
+  {
+    RuleFor(q => q.Content)
+      .MaximumLength(300)
+      .WithMessage("Content is too long.");
+
+    When(q => q.Deadline.HasValue, () =>
+    {
+      RuleFor(q => q.Deadline)
+        .Must(d => d > DateTime.UtcNow.AddDays(1).AddSeconds(2))
+        .WithMessage("The deadline must be at least 24 hours from now.");
+    });
+
+    When(q => q.GroupId.HasValue, () =>
```

Comment:
I think this is not necessary, because if we don't have the ID, we don't do anything with it, if we get it, we make request the database and if there is nothing there, we send an error messege. Adding this check will only reduce the speed of the program

Reply:
I'm not agree, but OK, it's you PR.
Then add to error message, that problem can be with ID too